test/uk/gov/hmrc/agentsubscriptionfrontend/validators/CommonValidatorsSpec.scala

```diff
@@ -485,12 +490,12 @@ class CommonValidatorsSpec extends UnitSpec with EitherValues {
       }
     }
 
-    "accumulate errors if there are multiple validation problems" in {
+    s"accumulate errors if there are multiple validation problems for addressline $lineNumber" in {
       val tooLongAndNonMatchingLine = "123456789012345678901234567890123456<"
       bind(tooLongAndNonMatchingLine) shouldBe Left(
         List(
-          FormError("testKey", "error.address.lines.maxLength", Seq(35)),
-          FormError("testKey", "error.address.lines.invalid", Seq())))
+          FormError("testKey", s"error.addressline.$lineNumber.maxlength", Seq(35)),
+          FormError("testKey", s"error.addressline.$lineNumber.invalid", Seq())))
```

Comment:
are multiple errors visible for each line when submitted? is this a visual requirement? just make sure

Reply:
Yes, we are showing multiple errors for same field. This was the behaviour before. Please ask Ian if its not fine.